samples/client/petstore/typescript-fetch/builds/es6-target/apis/PetApi.ts

```diff
@@ -208,16 +207,12 @@ export class PetApi extends runtime.BaseAPI {
             queryParameters['tags'] = requestParameters.tags.join(runtime.COLLECTION_FORMATS["csv"]);
         }
 
-        if (requestParameters.maxCount !== undefined) {
-            queryParameters['maxCount'] = requestParameters.maxCount;
-        }
-
         const headerParameters: runtime.HTTPHeaders = {};
 
         if (this.configuration && this.configuration.accessToken) {
             // oauth required
             if (typeof this.configuration.accessToken === 'function') {
-                headerParameters["Authorization"] = this.configuration.accessToken("petstore_auth", ["read:pets"]);
+                headerParameters["Authorization"] = this.configuration.accessToken("petstore_auth", ["write:pets", "read:pets"]);
```

Comment:
why does this change?

Reply:
It would be due to a fix in oauth scope to propagate these values probably.